src/View/SSViewer.php

```diff
@@ -822,17 +895,29 @@ public function setTemplateFile($type, $file)
      * @param string $contentGeneratedSoFar The content of the template generated so far; it should contain
      * the DOCTYPE declaration.
      * @return string
+     * @deprecated 5.4.0 Use getBaseTag() instead
      */
     public static function get_base_tag($contentGeneratedSoFar)
+    {
+        Deprecation::withSuppressedNotice(fn() => Deprecation::notice('5.4.0', 'Use getBaseTag() instead'));
+        // Is the document XHTML?
+        $isXhtml = preg_match('/<!DOCTYPE[^>]+xhtml/i', $contentGeneratedSoFar ?? '');
+        return static::getBaseTag($isXhtml);
+    }
+
+    /**
+     * Return an appropriate base tag for the given template.
+     * It will be closed on an XHTML document, and unclosed on an HTML document.
+     *
+     * @param bool $isXhtml Whether the DOCTYPE is xhtml or not.
+     */
+    public static function getBaseTag(bool $isXhtml = false): string
```

Comment:
For simple methods like this it makes sense to implement in 5.4 so people can swap to it directly if they were calling it in their code.